sci-electronics/kicad/kicad-4.0.5-r1.ebuild

```diff
@@ -66,14 +66,15 @@ src_prepare() {
 	# Patch to work with >=boost 1.61
 	eapply "${FILESDIR}/${PN}-boost-1.61.patch"
 
-	# remove all the non unix file endings
+	# Remove cvpcb desktop file as it does nothing
+	rm "${WORKDIR}/${P}/resources/linux/mime/applications/cvpcb.desktop" || die
```

Comment:
The whole of Gentoo uses relative paths and has been absolutely fine. Any modification of real system files would be caught by the sandbox. Please, just do like the rest of Gentoo and use relative paths. Anyhow, the absolute path is even worse, because you're implictly saying that `${S} == ${WORKDIR}/${P}` which is not guaranteed in general.